Source/ACE/Entity/Door.cs

```diff
@@ -320,15 +291,20 @@ private void Close(ObjectGuid closer = new ObjectGuid())
 
         private void Reset()
         {
-            if ((Time.GetTimestamp() - UseTimestamp) < ResetInterval)
-                return;
+            ActionChain chain = new ActionChain();
```

Comment:
Reset is already called from an ActionChain on "this", this action chain is redundant.